### ISSUE TITLE
Collaborators Needed

### ISSUE THREAD

**Issue by markusmoenig** [markusmoenig/Eldiron#29]
I think what I need for a v1 are some collaborators who can help with the design of the character system and help to flesh out the demo. I am mostly a developer and while I played the Ultima and similar games a lot, I am not an expert in all of the character class design systems on the market. I just want to make sure to support all possible use cases for weapons, gear, spells, crafting character progression and so on, which is a lot of stuff. 

Would be great if some of you would collaborate on this a bit, it would make a v1 much easier todo on my side as I do not need to worry about design choices all the time.

**Comment by ValorZard** [comment]
Have you check out RPG Maker? It's a similar game engine to what your making.

**Comment by markusmoenig** [comment]
Yeah. But I do my own one for fun 😅 So ? RPGMaker is also not open source.

**Comment by ValorZard** [comment]
So could you make a JPRG style game with this engine?

**Comment by markusmoenig** [comment]
You could use any game which uses square tiles, however not every area of the engine will be as worked out as RPG Maker yet, like for example the UI gets created via scripts right now.

**Comment by freewu32** [comment]
Maybe making your to-do list is a great first step

**Comment by markusmoenig** [comment]
@freewu32 Yes, good idea. But with the Creator rewrite everything is a bit in a state of flux right now.

I will do a roadmap once I get a grip on things again :)

**Comment by coolcoder613eb** [comment]
I would like to help, where can I start?
(Mostly not code, as I am still learning Rust 😉)

**Comment by markusmoenig** [comment]
@coolcoder613eb Thanks for the offer. I am currently rewriting the Creator and once finished there will be quite a few things todo except coding. I will write a message here once ready with the list (to see if something fits).

**Comment by coolcoder613eb** [comment]
Maybe you could show the status of the Creator rewrite in the README?

**Comment by markusmoenig** [comment]
Tis is the current state, progress is accelerating as the new UI framework is getting more complete.

<img width="1312" alt="Screenshot 2023-11-23 at 20 46 53" src="https://github.com/markusmoenig/Eldiron/assets/10293923/21cb3ce2-b29e-4c29-a59d-4a78d2cd284c">

**Comment by coolcoder613eb** [comment]
I meant an issue with a task list or something.

Is there any non-video tutorials for Eldiron?
I was trying to make something, but I didn't know how to use it. 😞

**Comment by coolcoder613eb** [comment]
(It just struck me, most large rust GUI projects seem to roll their own GUI toolkit 😉)

**Comment by markusmoenig** [comment]
No, not yet, I documented the old version at Eldiron.com but the new version has nothing yet, just wait a bit until the new version is up and running and I write tutorials.

GUI toolkits, the problem is that every app has different requirements, its hard todo a "one fits all" kind of toolkit.

**Comment by coolcoder613eb** [comment]
A (simple) walkthrough is a ***\*MUST\****.
Just going through opening up the editor, making a map, a enemy/npc, and a win.
